packages/gp2-components/src/templates/OutputForm.tsx

```diff
@@ -75,6 +91,11 @@ type OutputFormType = {
     typeof AuthorSelect
   >['loadOptions'];
   keywordSuggestions: gp2Model.TagDataObject[];
+  getRelatedOutputSuggestions?: NonNullable<
```

Comment:
Does it give an error somewhere if we make this props required?
```suggestion
  getRelatedOutputSuggestions: NonNullable<
```
I saw an error in CRN once where we forgot to pass the suggestion function to Working Groups but we passed to Teams but the card appeared in both so it was working for Teams but not for Working Groups. And if the suggestion function was required in the form we would find this error with Typescript